### DISCUSSION TITLE
2005/05/20/how-to-create-message-board

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#30] [Announcements]
# 2005/05/20/how-to-create-message-board

Step1，Create a new entry for the message board。

https://blog.qizhen.xyz/2005/05/20/how-to-create-message-board

**Comment by ruanqizhen** [comment]
《How to create message board.》有5条评论
东
2005年11月26日 15:11 编辑
xiexie

回复
健
2005年07月31日 23:27 编辑
Didn\’t be necessary, don\’t learn.

回复
艳沁
2005年05月25日 10:58 编辑
按照你的方法一步步做了,怎么到了最后一步就不对了呢?

回复
Chris
2005年05月22日 15:22 编辑
嘿嘿~~被你做成E文版了~~

回复
倩
2005年05月20日 18:41 编辑
正好在做留言板呢，哈，３Ｑ
